server/services/licenceService.ts

```diff
@@ -105,8 +104,19 @@ export default class LicenceService {
       standardPssConditions: [LicenceType.PSS, LicenceType.AP_PSS].includes(licenceType)
         ? getStandardConditions(LicenceType.PSS)
         : [],
+      responsibleComStaffId: responsibleOfficer.staffId,
     } as CreateLicenceRequest
 
+    // TODO: This section can be removed after having been live in production for som time. This is only needed initially because some
+    //  COM records will not be saved in our database initially. Over time, the OFFENDER_MANAGER_CHANGED event, and logins will have populated
+    //  staff details into the database, and this call will have become redundant
```

Comment:
Initially, our `community_offender_manager` table will be empty, and will gradually be populated by the `OFFENDER_MANAGER_CHANGED` event. However, we cannot be sure initially that when a COM creates a licence on behalf of a team mate, that the responsible COM details are in the database because they may not have logged into CVL, and they may not have had a case newly allocated to them since the service is moved to production. After a while on live, I suspect this section can be removed

Reply:
This answers one of the questions from the API review.